app/src/components/widgets/multiLevelWidgets/mostPopularPatterns/mostPopularPatterns.jsx

```diff
@@ -160,8 +160,14 @@ export class MostPopularPatterns extends Component {
     return (
       <div className={cx('popular-patterns')}>
         <div className={cx('attribute-selector')}>
-          <div className={cx('attribute-label')}>{attributeKey}</div>
+          <div className={cx('attribute-key')}>
+            <span className={cx('description')}>KEY:</span>
```

Comment:
I suppose that these words need to be translated as we translate key and value terms in other cases (see the screenshot below). You can add it to the `COMMON_LOCALE_KEYS` as an option.

![image](https://user-images.githubusercontent.com/31861805/217002250-009fa007-9c63-4b81-93ce-e08e33a26a4c.png)